packages/dht/src/dht/store/StoreRpcLocal.ts

```diff
@@ -29,10 +30,14 @@ export class StoreRpcLocal implements IStoreRpc {
         this.config = config
     }
 
+    private isLocalNodeStorer(dataKey: DhtAddress): boolean {
+        return this.config.getStorers(dataKey).some((p) => areEqualPeerDescriptors(p, this.config.localPeerDescriptor))    
+    }
+
```

Comment:
Could move this to to the end of the file as this is a utility function used by other methods